admin/includes/css/layout_controller.css

```diff
@@ -0,0 +1,94 @@
+#lbc-container {
+    font-size: 13px;
+}
+.d-none {
+    display: none;
+}
+.ui-sortable-placeholder {
+    visibility: visible!important;
+    border: 1px solid #31708f;
+}
+#lbc-main ul > li:hover {
+    color: #31708f;
+    background-color: #d9edf7;
+    border-color: #bce8f1;
+}
+li.lbc-item {
+    padding: 5px 10px;
+}
+.border-dark {
+    border-color: #343a40;
+}
+i.fa-angle-up:hover,
+i.fa-angle-down:hover,
+i.fa-xmark:hover,
+.panel-collapse:hover {
+    cursor: pointer;
+}
+.d-flex {
+    display: flex;
+}
+.justify-content-around {
+    justify-content: space-around;
+}
+.mb-0 {
+    margin-bottom: 0;
+}
```

Comment:
Should these BS4/5 styles be moved to the core stylesheet?
There are several other bridging styles there already.

Reply:
We can do that in a separate PR.